listening-platform/src/main/kotlin/com/nikitahohulia/listeningplatform/service/UserServiceImpl.kt

```diff
@@ -98,6 +107,7 @@ class UserServiceImpl(
             .flatMap { (user, publisher) ->
                 user.subscriptions.add(publisher.id!!)
                 userRepository.save(user)
+                redisUserRepository.update(user)
```

Comment:
```kotlin
                userRepository.save(user)
                redisUserRepository.update(user)
```

As I understand, the both invocations are reactive. 
So, with your changes `userRepository.save(user)` is not subscribed. I bet IDEA warn you about this